src/frontend/src/lib/components/tokens/TokenCardWithUrl.svelte

```diff
@@ -5,6 +5,7 @@
 	import { i18n } from '$lib/stores/i18n.store';
 
 	export let token: Token;
+	export let disableTabSelector: boolean = false;
```

Comment:
Linter does not complain about `: boolean` being not necessary?

Reply:
No, but I can introduce the ESlint rule

Reply:
Weird, I asked because I thought it was the default eslint configuration of SvelteKit projects.